e2e/__tests__/__snapshots__/showConfig.test.ts.snap

```diff
@@ -35,7 +35,6 @@ exports[`--showConfig outputs config info and exits 1`] = `
       ],
       "moduleNameMapper": [],
       "modulePathIgnorePatterns": [],
-      "name": "[md5 hash]",
```

Comment:
should `id` exist here now that I have renamed `name` to `id`?

Reply:
yes